Examples/Device/Device.SmallMemory.nanoFramework/app.config

```diff
@@ -0,0 +1,11 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
We don't support app.config for assembly binding. If VS added this one here, something is wrong with a referenced assembly.

Reply:
Removed